.circleci/config.yml

```diff
@@ -120,7 +120,7 @@ jobs:
           no_output_timeout: 10m
       - run:
           name: Undo all migrations
-          command: for m in `git diff --name-only HEAD..origin/master | grep "db/migrations/"`; do if ! make db-migrate-undo; then break; fi; done
+          command: for m in db/migrations/*.sql; do if ! make db-migrate-undo; then exit 1; fi; done
```

Comment:
Fix one thing at a time.  Here you both fix a real bug, and revert my change. Useless to work in this PR is the commit revert the change.